src/test/resources/org/opa/ideaplugin/lang/parser/fixtures/object_comprehension.rego

```diff
@@ -9,10 +9,3 @@ app_to_hostnames := {app.name: hostnames |
                     s.name == name
                     hostname := s.hostname]
 }
-
-
-
-merge_objects(a, b) = c {
```

Comment:
Why did you delete this case?

Reply:
This is not an object comprehension, it's a set comprehension. So a different case.

Reply:
Shouldn't we move it to appropriate file then?

Reply:
I moved it to a separate test case